packages/core/src/lib/export-import/import/import.service.ts

```
@@ -1274,894 +354,55 @@
-					{ joinTableName: 'tag_warehouse' }
-				]
-			},
-			...this.dynamicEntitiesClassMap
-		] as IRepositoryModel<any>[];
+				} catch (error) {
+					console.log(chalk.red('Failed to add the current user to imported organization', error));
+					reject(error);
+				}
+				resolve(true);
+			});
+		});
```

Biome (1.9.4)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)